lib/client.js

```diff
@@ -59,7 +59,7 @@ import { RealtimeUtils } from './utils.js';
 /**
  * @typedef {Object} InputAudioContentType
  * @property {"input_audio"} type
- * @property {string} [audio] base64-encoded audio data
```

Comment:
later we check if ArrayBuffer | Int16Array, and if so, we convert to base64.  Here we ensure the type allows for ArrayBuffer | Int16Array